packages/iov-tendermint-rpc/src/v0-20/requests.ts

```diff
@@ -31,6 +32,16 @@ export class Params extends requests.DefaultParams {
     return jsonRpcWith(req.method, req.params);
   }
 
+  public static encodeSubscribe(
+    req: requests.SubscribeRequest,
+    query: requests.SubscribeRequestQuery | undefined,
+  ): JsonRpcRequest {
+    const allTags: ReadonlyArray<Tag> = [{ key: "tm.event", value: req.type }, ...(query ? query.tags : [])];
+
+    const queryString = requests.buildTxQuery({ tags: allTags });
```

Comment:
I like how you make the `tm.event` fit alongside other tags. Nice unification.

Does `requests.buildTxQuery` make sense as is or needs to be modified?

Reply:
Makes perfect sense as the subscription type is just another tag (e.g. `tm.event='Tx'` or `tm.event='Tx' AND app.creator='jae'`). Also the Tendermint documentation states that those subscription queries are supposed to work like searches.